### ISSUE TITLE
Moviepy Image Module

### ISSUE THREAD

**Issue by Zayn-Rekhi** [Zulko/moviepy#1000]
So basically I am using the Pillow module in python and also the `moviepy` library in order to make a video out of pictures. Though, when I try to get the images using `moviepy` through the `my_clip = Image(f"{SmallVidCount}.jpg")`, the `Image() i`s read as something from `PIL` library. It gives me `the object has no attribute error` in python.

I have tried to use `mpe.Image(f"{SmallVidCount}.jpg")`, but it gives me the module `'moviepy.editor' has no attribute 'Image`' error. The First one is `module 'object' has no attribute 'Image'`. 

                    my_clip = mpe.Image(f"{SmallVidCount}.jpg") # has infinite duration
    				my_clip.set_duration(path_lengthList[SmallVidCount]).write_videofile("f{SmallVidCount}.mp4") # works !
    				myclip.set_audio(f"{SmallVidCount}.mp3")
    				SmallVidCount+=1

I just want it to output a video that is made out of the pictures. I got this code from [https://zulko.github.io/moviepy/getting_started/videoclips.html][1]

  [1]: https://zulko.github.io/moviepy/getting_started/videoclips.html

**Comment by Zayn-Rekhi** [comment]
```
 my_clip = mpe.Image(f"{SmallVidCount}.jpg") # has infinite duration
				my_clip.set_duration(path_lengthList[SmallVidCount]).write_videofile("f{SmallVidCount}.mp4") # works !
				myclip.set_audio(f"{SmallVidCount}.mp3")
				SmallVidCount+=1
```

**Comment by tburrows13** [comment]
Use `ImageClip`, not `Image`.

**Comment by tburrows13** [comment]
Thanks for reporting this, I've fixed the faulty docs in #1159.
